lib/cc_engine/issue.rb

```diff
@@ -1,26 +1,50 @@
-require "virtus"
+# typed: strict
 require "json"
 
 module CCEngine
   class Issue
-    include Virtus.model(strict: true)
+    extend T::Sig
 
-    attribute :check_name, String
-    attribute :description, String
-    attribute :categories, Array[String]
-    attribute :location
-    attribute :remediation_points
-    attribute :content
-    attribute :fingerprint
+    sig {
+      params(
+        check_name: String,
+        description: String,
+        categories: T::Array[String],
+        location: T.any(CCEngine::Location::LineRange, CCEngine::Location::Position),
+        remediation_points: T.nilable(Integer),
+        content: T.nilable(String),
+        fingerprint: T.nilable(String)
+      ).void
+    }
+    def initialize(
+      check_name:,
+      description:,
+      categories:,
+      location:,
+      remediation_points:,
+      content:,
+      fingerprint:
+    )
+      @check_name = check_name
+      @description = description
+      @categories = categories
+      @location = location
+      @remediation_points = remediation_points
+      @content = content
+      @fingerprint = fingerprint
+    end
```

Comment:
No idea what Virtus is doing, but this looks really similar to what `T::Struct` does. You might want to take a look and see if it fits your needs if you haven't already.